packages/frontend-2/components/header/NavBar.vue

```diff
@@ -3,7 +3,10 @@
     <nav class="fixed z-20 top-0 h-14 bg-foundation shadow hover:shadow-md transition">
       <div class="flex items-center justify-between h-full w-screen px-4">
         <div class="flex items-center truncate">
-          <HeaderLogoBlock :active="false" />
+          <HeaderLogoBlock
+            :active="false"
+            :to="activeUser ? '/' : 'https://speckle.systems/'"
```

Comment:
Wouldn't the 2nd URL be dependant on the environment? Do we always want to send guests to speckle.systems even if they're on a completely different deployment like latest.speckle.systems or even an academy one?

Reply:
Good point, @benjaminvo what are your thoughts?

Reply:
Yeah good question. Yes it should be dependent on the environment